community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacade.java

```diff
@@ -84,25 +88,26 @@
 import org.neo4j.kernel.lifecycle.LifecycleException;
 import org.neo4j.kernel.security.URLAccessValidationError;
 import org.neo4j.logging.Log;
-import org.neo4j.tooling.GlobalGraphOperations;
 
 import static java.lang.String.format;
-
 import static org.neo4j.collection.primitive.PrimitiveLongCollections.map;
+import static org.neo4j.graphdb.DynamicLabel.label;
 import static org.neo4j.helpers.collection.IteratorUtil.emptyIterator;
+import static org.neo4j.kernel.api.CountsRead.ANY_LABEL;
 import static org.neo4j.kernel.impl.api.operations.KeyReadOperations.NO_SUCH_LABEL;
 import static org.neo4j.kernel.impl.api.operations.KeyReadOperations.NO_SUCH_PROPERTY_KEY;
 
 /**
  * Implementation of the GraphDatabaseService/GraphDatabaseService interfaces. This delegates to the
  * services created by the {@link org.neo4j.kernel.impl.factory.GraphDatabaseFacadeFactory}.
- *
+ * <p/>
```

Comment:
As a side note: IntelliJ likes to insert these if language level 7 is selected for the project (or inferred from a pom.xml), presumably for backward compatibility. If language level 8 is set, it procudes a proper ```<p>``` tag. Unfortunately there's no switch to tell IntelliJ to use the language level 8 behaviour in all cases, so this might become a bit of an annoyance when switching back and forth between 2.3 and 3.0.